src/Command/SendCreditMemo.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Command;
+
+final class SendCreditMemo
+{
+    private $id;
+
+    public function __construct($id)
+    {
+        $this->id = $id;
+    }
+
+    public function Id(): string
```

Comment:
`Id()` -> `id()`

Reply:
And shouldn't the id be integer?

Reply:
No, CreditMemo Entity has id with string type.
![Screenshot 2019-05-14 at 10 33 53](https://user-images.githubusercontent.com/29897151/57683315-b0046b00-7633-11e9-9532-84071972b66a.png)